modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -1863,7 +1867,13 @@ open class PlaybackManager @Inject constructor(
         PLAYER("player"),
         NOTIFICATION("notification"),
         FULL_SCREEN_VIDEO("full_screen_video"),
+        UP_NEXT("up_next"),
         MEDIA_BUTTON_BROADCAST_ACTION("media_button_broadcast_action"),
+        MEDIA_BUTTON_BROADCAST_SEARCH_ACTION("media_button_broadcast_search_action"),
+        PLAYER_BROADCAST_ACTION("player_broadcast_action"),
```

Comment:
I added `player_broadcast_action` source to track any event fired from `PlayerBroadcastReceiver`. I wasn't able to receive any event in that receiver. Seems like those events are now received in `MediaSessionManager.onMediaButtonEvent(...)` and tracked with the source `media_button_broadcast_action`. I still thought to add this source to be sure we're not missing it.